doc/api/stream.md

```diff
@@ -2914,6 +2914,19 @@ Calling `Readable.from(string)` or `Readable.from(buffer)` will not have
 the strings or buffers be iterated to match the other streams semantics
 for performance reasons.
 
+If an `Iterable` object containing promises is passed as an argument,
+it could result in uncaught exceptions.
```

Comment:
```suggestion
it might result in uncaught exceptions.
```

Ref: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may ("Use *might* to express possibility.")